docs/en/latest/concepts/annotations.md

```diff
@@ -1,133 +0,0 @@
----
```

Comment:
Why do you want to delete these documents?

Reply:
> Can you resolve the conflicts? Thanks!

Sorry, I'm afraid I don't know how to operate it. Can you help me?

Reply:
@sober-wang If you need some help, please ping me.  I have too much news, easy to miss.